common/test/src/IO/EntityModelTest.cpp

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
I'd prefer it if the fix is in the same commit as the test case so that we don't have commits where the tests don't run. No need to change this for this PR, just something I think we should keep in mind for the future. Previously it wasn't important because everything was squashed before merging, but we agreed that we might merge some PRs without squashing, and in such a case I think it's better not to have random commits where tests are failing, because that makes bisecting harder.

Reply:
Sure, agreed. 
Actually we should rebase instead of squash the PR, since the picking fixes have nothing to do with the entity rotation changes, so I can do that once we're happy with the PR.

Reply:
Ok, cool. Then I suggest to squash these two commits.